src/engine/controls/enginecontrol.h

```diff
@@ -48,17 +48,17 @@ class EngineControl : public QObject {
             UserSettingsPointer pConfig);
     ~EngineControl() override;
 
-    // Called by EngineBuffer::process every latency period. See the above
-    // comments for information about guarantees that hold during this call. An
-    // EngineControl can perform any upkeep operations that are necessary during
-    // this call.
+    /// Called by EngineBuffer::process every latency period.
+    /// See the above comments for information about guarantees that hold during this call.
+    /// An EngineControl can perform any upkeep operations necessary here.
+    /// @param dRate current playback rate in audio frames per second
```

Comment:
This is a relative value. It indicates how many samples of the track are consumed compared to the output range of Mixxx. The engine can make use of some renamings to avoid this confusion.

Reply:
What about:

- playRate
- playbackRate
- relativePlayrate

Reply:
In the CO interface we have "rate" as the rate slider, a scalable relative adjustment with 0 at unity. And 
https://github.com/mixxxdj/mixxx/blob/d61749c79c2ed2fd8e85cc86add5e3f58b600925/src/engine/controls/ratecontrol.cpp#L45

"rate_ratio" the fixed ratio relative to the track original rate with 1 at unity. 

Here it is the same relative to the engine rate.

einigeRateRatio  or such. However just "rate" is also ok. I was more concerned to the namings in EngineBuffer leading to this confusion.

Reply:
I moved the topic about renaming "rate" to a dedicated issue #13833 and hide this discussion here